Reportedly/AppDelegate.swift

```diff
@@ -31,11 +31,23 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
         LocalNotificationScheduler.shared.setupDailyStandupLocalNotifications()
         
-        // Then, show the login screen
         let window = UIWindow(frame: UIScreen.main.bounds)
         self.window = window
-        let module = LoginEmailModule()
-        module.router.show(on: window)
+        
+        if let userToken = UserDefaults.userToken,
+           let userId = UserDefaults.userId,
+           let userEmail = UserDefaults.userEmail,
+           let userSlackId = UserDefaults.userSlackId
+        {
```

Comment:
Can we wrap userToken and userSlackId into UserCredential model and userId, userEmail into UserInfo or something like that? What do you think? 😅

Reply:
yep sure, back then I am doing it quick for demoing, let me refactor here a bit 🙏